modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcThinFeature.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.odbc.jdbc;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import org.apache.ignite.internal.ThinProtocolFeature;
+
+/**
+ * Defines supported features for JDBC thin client.
+ */
+public enum JdbcThinFeature implements ThinProtocolFeature {
```

Comment:
Would all other thin protocols have their own `XXXThinFeature` subclasses? What about .NET features?

Reply:
There is not .net features.
ODBC and thin client will be implemented on separate tickets.